ports/psoc6/help.c

```diff
@@ -33,16 +33,17 @@ const char psoc6_help_text[] =
     "are in the 'psoc6' module.\n"
     "\n"
     "Quick overview of some objects:\n"
```

Comment:
Whether we should have this?

Reply:
Probably not, discussion in @NikhitaR-IFX same question.